packages/react-scripts/template/README.md

```diff
@@ -425,11 +425,11 @@ As a final step, you may find it convenient to run `watch-css` automatically wit
 
 Now running `npm start` and `npm run build` also builds Sass files. Note that `node-sass` seems to have an [issue recognizing newly created files on some systems](https://github.com/sass/node-sass/issues/1891) so you might need to restart the watcher when you create a file until it’s resolved.
 
-## Adding Images and Fonts
+## Adding Images, Fonts and Files
 
 With Webpack, using static assets like images and fonts works similarly to CSS.
 
-You can **`import` an image right in a JavaScript module**. This tells Webpack to include that image in the bundle. Unlike CSS imports, importing an image or a font gives you a string value. This value is the final image path you can reference in your code.
+You can **`import` a file right in a JavaScript module**. This tells Webpack to include that file in the bundle. Unlike CSS imports, importing a file gives you a string value. This value is the final path you can reference in your code, e.g. as the `src` attribute of an image or the `href` of a link to a PDF. For images less than 10,000 bytes, importing them will return a `data:` URL instead of a path, so that they are inlined instead of requiring an extra request to the server to fetch them. To be clear, importing any file with an unknown extension will return the final path to that file.
```

Comment:
Might be able to reduce it a bit more, with something like:

> Unlike CSS imports, importing a file returns a URL that can be used in your code, e.g. as the `src` attribute of an image or the `href` of a link to a PDF.
>
> To reduce the number of requests to the server, importing images that are less than 10,000 bytes return a [data URI]() instead of a path.